app/controllers/schools_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SchoolsController < ApplicationController
+  before_action :redirect_to_dashboard,
```

Comment:
I'd much prefer to see this `include DFEAuthentication`, then the following line becomes `if: :user_signed_in?`

Just DRYs up the definition of a signed in user.

Reply:
Yep, agree. Thought `DFEAuthentication` also did the redirect so checked manually, fixed.